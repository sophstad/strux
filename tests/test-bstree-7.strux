int main() {
    BSTree::int tree = new BSTree::int(1,3,2,4,6,0,-1,53);
    tree.show();
    return 0;
}