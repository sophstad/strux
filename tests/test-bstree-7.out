            (0001)                                                              
         *----*------------*                                                    
      (0000)            (0003)                                                  
   *-----*           *----*------*                                              
(-001)            (0002)      (0004)                                            
                                *------*                                        
                                    (0006)                                      
                                      *------*                                  
                                          (0053)                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
